Sources/RadioGroup.swift

```diff
@@ -14,21 +14,39 @@ import UIKit
         self.titles = titles
     }
 
+    public convenience init(attributedTitles: [NSAttributedString]) {
+        self.init(frame: .zero)
+        self.attributedTitles = attributedTitles
+    }
+
     open var titles: [String] = [] {
         didSet {
             stackView.removeAllArrangedSubviews()
             stackView.addArrangedSubviews(titles.map { RadioGroupItem(title: $0, group: self) })
 
-            // update every forEachItem
-            selectedColor = { selectedColor }()
-            buttonSize = { buttonSize }()
-            itemSpacing = { itemSpacing }()
-            isButtonAfterTitle = { isButtonAfterTitle }()
-            titleAlignment = { titleAlignment }()
-            selectedIndex = { selectedIndex }()
+            updateProperties()
+        }
+    }
+
+    open var attributedTitles: [NSAttributedString] = [] {
+        didSet {
```

Comment:
This is nice, but creates a new problem:

If you set
`radioGroup.attributedTitles = someArray`
and then do
`print(radioGroup.titles[0])`
you get an out-of-bounds crash.

I think a possible solution is to change the two `didSet` to `get` and `set` that access the labels directly.
What do you think?

Reply:
Another solution:

```swift
open var attributedTitles: [NSAttributedString] = [] {
    didSet {
        titles = attributedTitles.map { $0.string }
        // change titleLabel.attributedText of each item
    }
```

Reply:
I've added `titles = attributedTitles.map { $0.string }` variant

Reply:
Thanks. But now we have the opposite problem: if you set the `titles` and then try to access `attributedTitles` then you get an error.

Do you want me to fix this?